website/views/pages/homepage.ejs

```diff
@@ -119,7 +119,7 @@
         <div purpose="features-text" class="text-left m-0 order-1 pl-0 pl-lg-4">
           <div>
             <h2 class="mb-4">Complete control over the queries you run and what you collect</h2>
-            <p>Adapt out-of-the-box policies, import packs from the community, or start from scratch.  Collect custom data just for your organization. Run live queries in Fleet's dashboard, or use the API to easily create your own dashboards.  Programmatically check device posture.  Run custom workflows any time Fleet confirms compliance issues or severe vulnerabilities actually installed on computers.</p>
+            <p>Adapt out-of-the-box policies, import packs from the community, or start from scratch. Run live queries from Fleet’s dashboard or use our API to easily build your own. Create custom workflows whenever Fleet confirms compliance issues or detects vulnerable devices.</p>
```

Comment:
1. I'm not sure about "our" pronouns here, unless it's "our" in the sense of "all Fleet contributors and users, including you, dear reader"
2. You create the workflows ahead of time, but the workflows run whenever Fleet detects things
3. "severe vulnerabilities that are actually installed on computers" -- the phrase "actually installed on computers" here is a differentiator from how some enterprises work today (i.e. dealing with every vulnerability, whether or not it's actually installed anywhere)